plugins/module_utils/net_map/networking_mapper.py

```diff
@@ -650,6 +667,32 @@ def __build_network_tool_common(
         return tool_type(*args_list)
 
 
+    @staticmethod
+    def __build_network_tool_route(
+        tool_net_def: networking_definition.SubnetBasedNetworkToolDefinition,
+        tool_type: typing.Type,
+    ) -> networking_env_definitions.MappedMultusNetworkConfig:
+        args_list = [
+            [
+                _map_host_network_range_to_output(ip_range)
+                for ip_range in tool_net_def.ranges_ipv4
+            ],
+            [
+                _map_host_network_range_to_output(ip_range)
+                for ip_range in tool_net_def.ranges_ipv6
+            ],
```

Comment:
You can create these two by calling `__build_network_tool_common`. A bit less to maintain.